test/test_configInit.py

```diff
@@ -772,9 +822,9 @@ def test_25a_celery_worker_config(self):
         Ensure that basic celery worker config is handled
         """
         ConfigurationInit(TESTCONF, "sample.json")
-        self.assertEqual(1, len(globals.g_celeryworkerkwargs))
-        self.assertTrue("autoscale" in globals.g_celeryworkerkwargs)
-        self.assertEqual("4,4", globals.g_celeryworkerkwargs['autoscale'])
+        self.assertEqual(1, len(globals.g_celery_worker_kwargs))
```

Comment:
Why are we all wanked up about "worker" if the term is remaining in use for celery?  Should these be changed as well?

Reply:
I am not going to change it for things related to third-party software, because the terminology should match for consistency and to avoid confusion. If the third-party code uses a term, our code that refers to it is going to use the same term. In our code that is self-referential, I'm making to required changes to keep things consistent at that level. But if celery has workers, and has functions called `worker()` (etc.), then any code that directly refers to it will use those terms.